### PR TITLE
Import identities documentation improvements

### PR DESCRIPTION
**Pull request by TannerGabriel**
## Related issue

No related issue

## Proposed changes

The import identities command documentation was hard to understand since there were no example credential files. There were also no references to the import identities command in the managing user identities documentation.

## Checklist

- [X] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [X] I have read the [security policy](../security/policy).
- [X] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [X] I have added tests that prove my fix is effective or that my feature
      works.
- [X] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: cmd/identities/import.go [hunk 1]
```diff
@@ -17,7 +17,17 @@ import (
 var ImportCmd = &cobra.Command{
 	Use:   "import <file.json [file-2.json [file-3.json] ...]>",
 	Short: "Import identities from files or STD_IN",
-	Example: `$ kratos identities import file.json
+	Example: `
+$ cat >./file.json <<EOF
+{
+    "schema_id": "default",
+    "traits": {
+        "email": "foo@ory.sh"
```
Comment (vinckr) [inline cmd/identities/import.go:25]:
```suggestion
        "email": "foo@example.com"
```
Maybe example.com is better, dont want to get spammed on ory.sh ;).

**Review by zepatrik — APPROVED** [review]
Nice, awesome :tada:

File: cmd/identities/import.go (hunk 1 above)
Comment (TannerGabriel) [reply cmd/identities/import.go]: Yeah sure changed it now!

**Review by vinckr — APPROVED** [review]

File: cmd/identities/import.go [hunk 2]
```diff
@@ -17,7 +17,17 @@ import (
 var ImportCmd = &cobra.Command{
 	Use:   "import <file.json [file-2.json [file-3.json] ...]>",
 	Short: "Import identities from files or STD_IN",
-	Example: `$ kratos identities import file.json
+	Example: `
+$ cat >./file.json <<EOF
```
Comment (aeneasr) [inline cmd/identities/import.go:21]:
```suggestion
	Example: `$ cat > ./file.json <<EOF
```

File: cmd/identities/import.go [hunk 3]
```diff
@@ -17,7 +17,17 @@ import (
...
+{
+    "schema_id": "default",
+    "traits": {
+        "email": "foo@example.com"
+    }
+}
+EOF
+
+$ kratos identities import file.json
 $ cat file.json | kratos identities import`,
```
Comment (aeneasr) [inline cmd/identities/import.go:31]:
```suggestion
# Alternatively:
$ cat file.json | kratos identities import`,
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by aeneasr** [comment]
Awesome, thank you!

**Pull request merged**